pennylane/devices/default_mixed.py

```diff
@@ -975,8 +976,24 @@ def execute(
         self,
         circuits: QuantumScript,
         execution_config: Optional[ExecutionConfig] = None,
-    ) -> None:
-        raise NotImplementedError
+    ) -> Union[Result, ResultBatch]:
+        interface = (
+            execution_config.interface
+            if execution_config.gradient_method in {"best", "backprop", None}
+            else None
+        )
```

Comment:
I feel that this is something that should've been handled in preprocess.

Reply:
fixed